src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -1,36 +1,84 @@
 package seedu.duke.commands;
 
+import seedu.duke.entries.Category;
 import seedu.duke.entries.Entry;
-import seedu.duke.entries.Food;
-import seedu.duke.entries.Income;
-import seedu.duke.entries.Other;
-// import seedu.duke.entrylog.EntryLog;
+import seedu.duke.entrylog.EntryLog;
 
-public class AddCommand {
+/**
+ * Represents the add feature in PocketPal. Users may provide a description
+ * and specify the corresponding price and category of their entry
+ * e.g., <code>/add lunch -p 15 -c food</code>
+ */
+public class AddCommand extends Command {
+    private static final String CATEGORY_CLOTHING = "clothing";
+    private static final String CATEGORY_ENTERTAINMENT = "entertainment";
     private static final String CATEGORY_FOOD = "food";
-    private static final String CATEGORY_OTHER = "other";
     private static final String CATEGORY_INCOME = "income";
+    private static final String CATEGORY_MEDICAL = "medical";
+    private static final String CATEGORY_OTHERS = "others";
+    private static final String CATEGORY_PERSONAL = "personal";
+    private static final String CATEGORY_TRANSPORTATION = "transportation";
+    private static final String CATEGORY_UTILITIES = "utilities";
```

Comment:
Please use the constants in [`EntryConstants.java`](https://github.com/AY2223S2-CS2113-W15-2/tp/blob/master/src/main/java/seedu/duke/constants/EntryConstants.java). Doing so will help reduce code clutter and better accommodate to future changes if required.